File: search/linear_search/python/linearsearch.py
arr=[10,2,3,4,23,5,21,45,90,100];
item=int(input("Enter the item which you want to search"));
for i in range(0,len(arr)):
  if arr[i]==item:
    flag=i+1;
    break;
  else:
    flag=0;
if falg!=0:
  print("Item found at location %d", flag));
else:
  print("Item not found");

File: search/linear_search/Matlab/linear_search.m
array = input('\nNumber of array:');

initializeArray = zeros(array, 1);

for loop = 1:array
    initializeArray(loop, 1) = input(sprintf('\nInput number [%d]:', loop));
end

item = input('\nEnter the number to be search:');

for loop = 1:array
    if(initializeArray(loop, 1) == item)
        fprintf('\nThe number is found in index: %d', loop)
        break
    end
end